src/History/Log.php

```diff
@@ -60,9 +62,30 @@ public function __construct(
         $this->consumerName = $consumerName;
     }
 
-    public function getId(): string
+    public function getUniqueKey(): string
     {
-        return $this->id;
+        return $this->id . '_' . $this->date->format('c');
+    }
+
+    public function withoutAuthor(): Log
+    {
+        $c = clone $this;
```

Comment:
I know it's nitpicky but even though this is a 3-line function I would still like if the variable has a name like `log` or `clone` than just `c` :)